src/main/resources/static/js/datastream.js

```diff
@@ -125,158 +125,158 @@ function createDS() {
     $.ajax({
         type: "POST",
         url: "datastream/create",
-        datatype: 'json',
-        contentType: 'application/json',
+        datatype: "json",
+        contentType: "application/json",
         data: JSON.stringify(mydata),
         error: function (response) {
             $.notify({
-                message: 'Datastream could not be created, check the Log for errors'
+                message: "Datastream could not be created, check the Log for errors"
             }, {
                 allow_dismiss: true,
-                type: 'danger',
+                type: "danger",
                 placement: {
                     from: "top",
                     align: "left"
                 },
                 animate: {
-                    enter: 'animated fadeInDown',
-                    exit: 'animated fadeOutUp'
+                    enter: "animated fadeInDown",
+                    exit: "animated fadeOutUp"
                 }
             });
             addToLog(response.responseText);
         },
         success: function (e) {
             $.notify({
-                message: 'Datastream created.'
+                message: "Datastream created."
             },{
                 allow_dismiss:true,
-                type: 'info',
+                type: "info",
                 placement: {
                     from: "top",
                     align: "left"
                 },
                 animate: {
-                    enter: 'animated fadeInDown',
-                    exit: 'animated fadeOutUp'
+                    enter: "animated fadeInDown",
+                    exit: "animated fadeOutUp"
                 }
             });
-            addToLog('Datastream created.');
-            closeModal('dialog');
+            addToLog("Datastream created.");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['addToLog' is not defined. (no-undef)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)